src/services/auth/AuthService.js

```
@@ -441,6 +477,38 @@ class AuthService {
           });
 
+          // Check if the error is due to an expired token (401 Unauthorized)
+          if (error.response && error.response.status === 401 && !originalRequest._retry) {
+            console.debug('Attempting to refresh token due to 401 Unauthorized error');
+
+            // Mark the request as retried to prevent infinite loops
+            originalRequest._retry = true;
+
+            try {
+              // Try to refresh the token
+              const refreshResponse = await this.refreshToken();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)